### PR TITLE
iphone-line-to-android-transfer: Add version 3.1.36

### PR DESCRIPTION
**Pull request by digrouz**

### CODE REVIEW

File: bucket/iphone-line-to-android-transfer.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            "url": "https://www.backuptrans.com/download/exe/iphone-line-to-android-transfer_x64.exe#/dl.7z",
+            "hash": "2513ac396d1318b3c16d6ee1b2dbe665aed258759b929296080dd27486be0254"
+        },
+        "32bit": {
+            "url": "https://www.backuptrans.com/download/exe/iphone-line-to-android-transfer.exe#/dl.7z",
+            "hash": "6302fc83933f0b4dafa547a00a9758f53276bddcc44b5c7d4ce6de4521cb1472"
+        }
+    },
+    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst*nsis\" -Recurse",
+    "notes": "Without iTunes application (https://support.apple.com/downloads/itunes) installed this is not possible to detect iPhone devices.",
```
Comment (Ash258) [inline bucket/iphone-line-to-android-transfer.json]:
```suggestion
    "notes": "Without iTunes application (https://support.apple.com/downloads/itunes) installed there is not possible to detect iPhone devices.",
```

File: bucket/iphone-line-to-android-transfer.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.backuptrans.com/download.html",
+        "regex": "(?sm)Backuptrans iPhone Line to Android Transfer.*?V([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://www.backuptrans.com/download/exe/iphone-line-to-transfer_x64.exe#/dl.7z"
```
Comment (Ash258) [inline bucket/iphone-line-to-android-transfer.json]:
```suggestion
                "url": "https://www.backuptrans.com/download/exe/iphone-line-to-android-transfer_x64.exe#/dl.7z"
```

**Comment by digrouz** [comment]
Hello, is there still any issue preventing the merge of this pull request?

**Comment by Ash258** [comment]
Trial/Freemium again. I am not going to merge them. After other collaborators decide how to handle these it will be merged.

**Comment by digrouz** [comment]
maybe creating another "non-free" bucket can be an option

**Comment by digrouz** [comment]
Do I have to open an issue or something to start the discussion ?

**Comment by linsui** [comment]
See https://github.com/lukesampson/scoop-extras/issues/2821

**Comment by digrouz** [comment]
is this issue liked with trial/freemium stuff ? I understood it about nightlies versions.

**Pull request closed without merging**